package.json

```diff
@@ -16,21 +17,31 @@
     "url": "https://github.com/freestarcapital/pubfig-adslot-react-component/issues"
   },
   "license": "ISC",
+  "peerDependencies": {
+    "react": "^15.7.0 || ^16.0",
+    "prop-types": "^15.7.2 "
+  },
   "dependencies": {
-    "@babel/runtime": "^7.12.5",
-    "@reach/router": "^1.3.4",
-    "lodash": "^4.17.20",
+    "lodash.isequal": "^4.5.0",
+    "lodash.sortby": "^4.7.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.12.1",
+    "@babel/plugin-transform-runtime": "^7.12.10",
+    "@babel/preset-env": "^7.12.11",
+    "@babel/preset-react": "^7.12.10",
+    "babel-loader": "^8.2.2",
+    "clean-webpack-plugin": "^3.0.0",
+    "np": "^7.2.0",
+    "path": "^0.12.7",
     "prop-types": "^15.7.2",
     "react": "^15.7.0",
-    "react-dom": "^15.7.0",
-    "react-scripts": "^3.4.1"
+    "webpack": "^5.15.0",
+    "webpack-cli": "^4.3.1"
   },
   "scripts": {
-    "start": "react-scripts start",
```

Comment:
do we not want to keep this, along with the demo files that were deleted for ease of development?

Reply:
agree on keeping the demo files if they are used for testing locally

Reply:
im creating a testing react app to test against locally as a private repo. Since this is a public repo we cant keep any real publisher names / fsdata in there without using env files which i dont think is worth doing to test locally compared to another repo to test with ( i just referenced the directory locally to test from )

Reply:
When we setup the testing framework here we can add that in lieu of this